.travis.yml

```diff
@@ -68,7 +64,7 @@ deploy:
     bucket: blf-deployments
     access_key_id: $AWS_ACCESS_KEY_ID
     secret_access_key: $AWS_SECRET_ACCESS_KEY
-    local_dir: dpl_cd_upload
+    local_dir: codedeploy_artefact
```

Comment:
The only reason this said `dpl_cd_upload` was because I copy/pasted it from [a tutorial](https://medium.com/@itsdavidthai/comprehensive-aws-ec2-deployment-with-travisci-guide-7cafa9c754fc) in 2017 🙈 

Nice to see it get a good name now!